### PR TITLE
AEA-1243 Updating readme files

### PR DESCRIPTION
**Pull request by 5A11**
This PR adds and updates packages' readme files

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/fetchai/contracts/erc1155/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Fetch ERC1155 Contract
+
+## Description
+
+This contract package is used to interface with a Fetch Oracle Client contract, which requests a real-world value from a Fetch Oracle contract.
```
Comment (DavidMinarsch) [inline packages/fetchai/contracts/erc1155/README.md:5]: This is not correct. The ERC1155 contract wraps a common standard, please link to it.

File: packages/fetchai/skills/confirmation_aw2/README.md [hunk 2]
```diff
@@ -10,11 +10,13 @@ This skill searches for an agent in a vicinity on the SOEF that sells the data t
...
 
 * `search`: searches for data selling service on SOEF
+* `transaction`: sequentially processes transactions' settlements on a blockchain
 
 ## Handlers
 
+* `default`: handles `default` messages for registration
 * `fipa`: handles `fipa` messages for negotiation
-* `ledger_api`: handles `ledger_api` messages for payment
+* `ledger_api`: handles `ledger_api` messages for processing transaction settlements
```
Comment (DavidMinarsch) [inline packages/fetchai/skills/confirmation_aw2/README.md:19]: ledger_api is generally for all interactions with ledger...

**Review by DavidMinarsch — COMMENTED** [review]
I believe every skill inheriting from generic buyer, as well as generic buyer itself has the transaction processing behaviour, please add.

File: packages/fetchai/skills/confirmation_aw2/README.md (hunk 2 above)
Comment (5A11) [reply packages/fetchai/skills/confirmation_aw2/README.md:19]: ok yes, I'll keep it generic

File: packages/fetchai/skills/carpark_client/README.md [hunk 3]
```diff
@@ -4,18 +4,18 @@
...
 This skill purchases information on available car parking spaces in a vicinity.
 
-This skill finds an agent on the SOEF which sells car park availability data in a vicinity, requests this data, negotiates the price, pays the proposed amount if agreement is reach, and receives the data bought.
+This skill finds an agent on the sOEF which sells car park availability data in a vicinity, requests this data, negotiates the price, pays the proposed amount if agreement is reach, and receives the data bought.
 
 
 ## Behaviours
 
-* search: searches for data selling service on SOEF
+* `search`: searches for car park data selling service on the sOEF
```
Comment (DavidMinarsch) [inline packages/fetchai/skills/carpark_client/README.md:12]: I think this is missing transaction behaviour

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**